src/neo/SmartContract/ContractTask.cs

```diff
@@ -0,0 +1,39 @@
+using System.Runtime.CompilerServices;
+
+namespace Neo.SmartContract
+{
+    [AsyncMethodBuilder(typeof(ContractTaskMethodBuilder))]
+    class ContractTask
+    {
+        private readonly ContractTaskAwaiter awaiter;
+
+        public static ContractTask CompletedTask { get; }
```

Comment:
should this be readonly?

Reply:
It is equivalent to `Task.CompletedTask`.